internal/http_client.go

```diff
@@ -140,7 +140,7 @@ func (r *Response) CheckStatus(want int) error {
 	if msg == "" {
 		msg = string(r.Body)
 	}
-	return fmt.Errorf("http error status: %d; reason: %s", r.Status, msg)
+	return fmt.Errorf("http error status: %d; reason: %q", r.Status, msg)
```

Comment:
This should not be quoted. It is breaking a bunch of tests now.

Reply:
yes , sorry I  was playing around with it, i though i removed you from review